userguide/dailyuse/libertine.rst

```diff
@@ -51,47 +57,51 @@ Destroy a container
 Manage applications
 -------------------
 
-Once a container is set up, the installed applications are available in the app drawer. You can also tap the name of the container in the *Manage Libertine Containers* menu.
+Once a container is set up, the installed apps are available in the app drawer.
+You can also tap the name of the container in the *Manage Libertine Containers* menu.
 
-You can also list them via command line::
+You can also list them via the command-line::
 
   libertine-container-manager list-apps
 
 Install a package
 ^^^^^^^^^^^^^^^^^
 
-From the apps list in the *Manage Libertine Containers* menu tap the + sign to get a dialog box to add a package.
+From the apps list in the *Manage Libertine Containers* menu you can tap the + sign to get a dialog box that lets you add a package.
 
-Enter the package name if you know it. Otherwise you can search the archives for a package. You can make this process easier by making the *Lubuntu Software Center* your first package install. Being a desktop app, you will need to use the *Libertine Tweak Tool* in the *Open Store* to make the text large enough to read on a mobile device.
+Enter the package name if you know it.
+Otherwise you can search the archives for a package.
+You can make this process easier by making the *Lubuntu Software Center* your first package install.
+Being a desktop app, you will need to use the *Libertine Tweak Tool* in the *Open Store* to make the text large enough to read on a mobile device.
 
-Packages may also be installed via command line::
+Packages may also be installed via the command-line::
 
   libertine-container-manager install-package -p PACKAGE-NAME
 
 Remove a package
 ^^^^^^^^^^^^^^^^
 
-On the package list, within the container (*System Settings* > *Libertine* > *Manage Libertine Containers* > *CONTAINER NAME*), swipe right on the package you want to remove to reveal the option to remove the package.
+On the package list, within the container (*System Settings* > *Libertine* > *Manage Libertine Containers* > *CONTAINER NAME*), swipe right on the package you want to remove to reveal the option to remove it.
```

Comment:
```suggestion
Remove a package by swiping it to the right from its entry in the package list (in *System Settings* > *Libertine* > *Manage Libertine Containers* > *CONTAINER NAME*). An option to remove it will be revealed.
```